service/src/main/resources/templates/ehr_observation_statement_template.mustache

```diff
@@ -6,11 +6,13 @@
             {{#effectiveTime}}<center value="{{effectiveTime}}"/>{{/effectiveTime}}{{^effectiveTime}}<center nullFlavor="UNK"/>{{/effectiveTime}}
         </effectiveTime>
         <availabilityTime value="{{availabilityTime}}" />
+        {{{value}}}
```

Comment:
If I'm reading the code right this is only used by ImmunizationObservationStatementMapper. I don't think Immunization has any structured value or reference range mappings.

You probably want to revert these template changes.

It would also be nice if you could rename this file to `immunization_observation_statement_template.mustache` to avoid any confusion in the future.